lib/worker.js

```diff
@@ -392,6 +394,10 @@ class AssetComputeWorker {
 
         rendition.eventSent = true;
 
+        const renditionSize = rendition.size();
+        // track total rendition size to add to activation metrics
+        this.totalRenditionsSize += renditionSize;
```

Comment:
Only note: For large renditions, if there are a lot (a lot) of them, this can overflow.

Reply:
true, I will create a safe add function that takes into account the max nodejs number: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER

fwiw, the max number is `9007199254740991`, which would translate to >9007 terabytes. I am pretty sure nui would timeout first if we were uploading that large amount of terabytes

Reply:
<img width="248" alt="buff-overflow" src="https://user-images.githubusercontent.com/20048485/154355932-38c44e37-f339-414e-9c1b-27edd310580c.png">

Reply:
Right, but adding the check won't hurt.

Reply:
It's for metrics, i don't think we need to go too far. If we have renditions this large we can always improve.

Reply:
Ok I will not add this check now, but we can always address it if we are seeing large amounts of renditions or very large renditions